spec/commands/create_invoice_spec.rb

```diff
@@ -50,4 +51,57 @@ def mock_price_on_satoshis
       expect { perform }.to raise_error("Invalid satoshi amount")
     end
   end
+
+  context "with 1 product" do
+    let(:products_hash) { { product_a_id => 1 } }
+    it "creates exactly 1 invoice product" do
```

Comment:
como detalle: antes de cada it, debiera ir una lína en blanco. ayuda a la lectura